src/main/resources/config.yml

```diff
@@ -60,6 +60,9 @@ displayLlama: true
 # Take parrots into account.
 displayParrot: true
 
+# Protect pets to avoid being hurt by other player.
+disablePlayerDamage: false
```

Comment:
Could you please add a new line in the `updateOldConfiguration` method of _PetMaster.java_ so that previous users have their configuration files updated?